tests/FoodStorage/test_foodStorage.py

```diff
@@ -13,28 +13,52 @@ def setUp(self):
 
     def testInitializeFoodStorage(self):
         self.assertFalse(self.foodStorage.isOpen)
-        self.assertEquals(dateToday, self.foodStorage.current_date)
+        self.assertEqual(dateToday, self.foodStorage.current_date)
         self.assertTrue(isEmptyInventory(self.foodStorage))
+        self.assertEqual(7, self.foodStorage.expiration_window.days)
+
+        self.foodStorage = FoodStorage(expiration_window=8)
+        self.assertFalse(self.foodStorage.isOpen)
+        self.assertEqual(dateToday, self.foodStorage.current_date)
+        self.assertTrue(isEmptyInventory(self.foodStorage))
+        self.assertEqual(8, self.foodStorage.expiration_window.days)
 
     def testAddFoods(self):
         # Inventory should start as empty
         self.assertTrue(isEmptyInventory(self.foodStorage))
 
         # Inventory should be sorted after adding multiple foods
-        addMultipleFoods(self.foodStorage, [food1, food2, food3])
-        self.assertEquals(
-            [(fooddate3, food3), (fooddate1, food1), (fooddate2, food2)], 
-            self.foodStorage.list()
-            )
+        self.foodStorage.addFoods([food1, food2, food3])
+        self.assertEqual([food3, food1, food2], self.foodStorage.list())
 
         # Adding new food, after listing, should still return a sorted inventory.
         self.foodStorage.addFoods([foodFuture, foodTomorrow, foodToday])
-        self.assertEquals(
-            [(fooddate3, food3), (fooddate1, food1), (fooddate2, food2), 
-            (dateToday, foodToday), (dateTomorrow, foodTomorrow), (dateOutsideRange, foodFuture)], 
+        self.assertEqual(
+            [food3, food1, food2, foodToday, foodTomorrow, foodFuture], 
             self.foodStorage.list()
             )
 
+    def testRemoveFoods(self):
+        # Inventory should start as empty
+        self.assertTrue(isEmptyInventory(self.foodStorage))
+
+        # Grab print output from calling list()
+        capturedListOutput = io.StringIO()
+        sys.stdout = capturedListOutput
+
+
+        self.foodStorage.remove(name1)
+
+        self.assertEqual(f"{REMOVE_FOOD_FAILURE_MESSAGE.format(name1)}\n", capturedListOutput.getvalue())
+        self.assertTrue(isEmptyInventory(self.foodStorage))
+
+        # reset standout
+        sys.stdout = sys.__stdout__
```

Comment:
What does resetting standout do?

Reply:
So earlier, on line 47, we temporarily set it to `capturedListOutput` so that we can verify our prints and this is just resetting it back to work normally.